### ISSUE TITLE
Translations doesn't apply to context menu in notification area

### ISSUE THREAD

**Issue by kubalav** [Flow-Launcher/Flow.Launcher#211]
Context menu in notification area contains 3 string (Open, Settings, Exit) in English only.

**Comment by taooceros** [comment]
Are you mean this one?
![image](https://user-images.githubusercontent.com/45326534/99473646-f0118000-2985-11eb-8a6d-c043d1cc5e48.png)

**Comment by kubalav** [comment]
See this:
![flow_launcher_context_menu_notification_area](https://user-images.githubusercontent.com/37414585/99494820-f564cf80-2971-11eb-99ee-9c30a95d3cc3.png)

**Comment by taooceros** [comment]
I have checked the code, and it seems because we changed the default language to the language in the setting later than the initialization of tray, and the ChangeLanguage method seems doesn't include the change for the name of the tray. Maybe this can be solved by adding changing resource to the tray name, since the ChangeLanguage method cannot be put earlier. I am not sure how to do that...... Let me investigate more.
